addons/mail/static/tests/helpers/mock_models.js

```diff
@@ -95,6 +95,14 @@ class MockModels {
                 },
                 records: [],
             },
+            'mail.channel_command': {
+                fields: {
+                    channel_types: { type: 'array' },
```

Comment:
type array is technically not existing in the server, so it shouldn't be in the mock server either. Misc type for fake field can be "binary" which supposedly let anything pass through.

Feel free to comment an array is expected, but don't mention it as type.